application/templates/backup.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if (.Values.backup).enabled }}
```

Comment:
```suggestion
{{- if .Values.backup.enabled }}
```
And add default value file with `backup.enabled: false` and some other sane default values if you want.

Reply:
Agreed @aslafy-z we should do that

Reply:
Added. Resolving comment